.github/workflows/release.yml

```diff
@@ -152,13 +201,80 @@ jobs:
+      if: startsWith(matrix.os, 'windows') && startsWith(matrix.runtime, 'win-arm64')
+      env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: https://uploads.github.com/repos/FreeTubeApp/FreeTube/releases/${{ secrets.UPLOAD_ID }}/assets{?name,label}
+        asset_name: freetube-${{ env.PACKAGE_VERSION }}-win-arm64-portable.zip
+        asset_path: build/freetube-${{ env.PACKAGE_VERSION }}-arm64-win.zip
+        asset_content_type: application/zip
+        
+    - name: Upload Windows x64 portable Release
+      uses: actions/upload-release-asset@v1
+      if: startsWith(matrix.os, 'windows') && startsWith(matrix.runtime, 'win-x64')
+      env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: https://uploads.github.com/repos/FreeTubeApp/FreeTube/releases/${{ secrets.UPLOAD_ID }}/assets{?name,label}
+        asset_name: freetube-${{ env.PACKAGE_VERSION }}-win-x64-portable.exe
+        asset_path: build/FreeTube ${{ env.PACKAGE_VERSION }}.exe
+        asset_content_type: application/x-ms-dos-executable
+
+    - name: Upload Windows arm64 portable Release
+      uses: actions/upload-release-asset@v1
+      if: startsWith(matrix.os, 'windows') && startsWith(matrix.runtime, 'win-arm64')
+      env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: https://uploads.github.com/repos/FreeTubeApp/FreeTube/releases/${{ secrets.UPLOAD_ID }}/assets{?name,label}
+        asset_name: freetube-${{ env.PACKAGE_VERSION }}-win-arm64-portable.exe
+        asset_path: build/FreeTube ${{ env.PACKAGE_VERSION }}.exe
+        asset_content_type: application/x-ms-dos-executable
+        
+    - name: Upload Mac x64 .dmg Release
       uses: actions/upload-release-asset@v1
-      if: startsWith(matrix.os, 'macos')
+      if: startsWith(matrix.os, 'macos') && startsWith(matrix.runtime, 'osx-x64')
       env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         upload_url: https://uploads.github.com/repos/FreeTubeApp/FreeTube/releases/${{ secrets.UPLOAD_ID }}/assets{?name,label}
-        asset_name: freetube-${{ env.PACKAGE_VERSION }}-mac.dmg
+        asset_name: freetube-${{ env.PACKAGE_VERSION }}-mac-x64.dmg
         asset_path: build/freetube-${{ env.PACKAGE_VERSION }}.dmg
         asset_content_type: application/x-apple-diskimage
+
+#    - name: Upload Mac arm64 .dmg Release
+#      uses: actions/upload-release-asset@v1
+#      if: startsWith(matrix.os, 'macos') && startsWith(matrix.runtime, 'osx-arm64')
+#      env:
+#          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#      with:
+#        upload_url: https://uploads.github.com/repos/FreeTubeApp/FreeTube/releases/${{ secrets.UPLOAD_ID }}/assets{?name,label}
+#        asset_name: freetube-${{ env.PACKAGE_VERSION }}-mac-arm64.dmg
+#        asset_path: build/freetube-${{ env.PACKAGE_VERSION }}-arm64.dmg
+#        asset_content_type: application/x-apple-diskimage
```

Comment:
My bad
This doesn't have to be commented due to `startsWith(matrix.runtime, 'osx-arm64')`
But it's also OK to stay commented, no need to update

Reply:
I'll leave this as-is. As it can be enabled easily and will prevent unnecessary action log entries